src/stim/simulators/matched_error.pybind.h

```diff
@@ -16,10 +16,38 @@
 #define _STIM_SIMULATORS_MATCHED_ERROR_PYBIND_H
 
 #include <pybind11/pybind11.h>
+#include "stim/simulators/matched_error.h"
+
 
 namespace stim_pybind {
 
-void pybind_matched_error(pybind11::module &m);
+pybind11::class_<stim::CircuitErrorLocationStackFrame> pybind_circuit_error_location_stack_frame(pybind11::module &m);
+pybind11::class_<stim::GateTargetWithCoords> pybind_gate_target_with_coords(pybind11::module &m);
+pybind11::class_<stim::DemTargetWithCoords> pybind_dem_target_with_coords(pybind11::module &m);
```

Comment:
Ooooouf, so many classes. Thanks for doing this one in particular.

Reply:
no worries!